### DISCUSSION TITLE
EDIT: Error in installing iso image in VMware Workstation

### DISCUSSION THREAD

**Discussion by nehahussain** [Security-Onion-Solutions/securityonion#8053] [Unsupported Versions] [answered]
I am downloading the iso image from here https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md but I am receiving an error that the iso image is corrupt.

**Comment by dougburks** [comment, marked as answer]
I've tested and it works correctly for me. If you continue to have problems, please try from a different computer or different network.

**Reply by nehahussain** [reply]
it is giving me error in installation in the last step.
"Install had a problem. please see /root/sosetup.log for details.
A summary of errors can be found in /root/errors.log."
![erro](https://user-images.githubusercontent.com/56157450/172106838-9ed63e37-f6e8-46a2-ab84-66ea35ec545c.PNG)

I checked the errors.log file and it shows me

**Comment by dougburks** [comment]
Instead of changing the title for a discussion that has already been marked as answered, please create a new discussion with an appropriate title.
